src/components/Tabs/Tabs.less

```diff
@@ -62,3 +106,19 @@
 		padding-top: 10px; // TODO: use our default gutter
 	}
 }
+
+.Tabs-mixin-tab-line(@color-Tab-line: @color-primary) {
```

Comment:
Just trying to keep it easy to read.  if they both are `@color` it might be a little confusing.